src/abstractdataframe/abstractdataframe.jl

```diff
@@ -992,6 +992,8 @@ Return a data frame containing only rows from `df` for which `fun` returns
 `true`.
 
 If `cols` is not specified then the predicate `fun` is passed `DataFrameRow`s.
+(Individual elements may be accessed by indexing the passed `DataFrameRow`s
+with column selectors inside the definition of `fun`.)
```

Comment:
Putting text in parentheses is weird. If we think this text is useful here it shouldn't hide it.
```suggestion
In that case, individual elements may be accessed by indexing the passed `DataFrameRow`s
with column selectors inside `fun`.
```

Reply:
I should probably mention the `row.x` possibility here too. How about the following?
"If `cols` is not specified then the predicate `fun` is passed `DataFrameRow`s.
Elements of a `DataFrameRow` may be accessed with dot syntax or column indexing inside `fun`."

Reply:
OK with me

Reply:
OK.